src/api.ts

```diff
@@ -17,11 +17,12 @@ import {
   TMP_DIR_PREFIX,
   canAuthGitRepo
 } from "./git";
-import {getPerforceManagerSingleton, IPerforceRepo, IPerforceView} from "./perforceManager";
+import {getPerforceManagerSingleton, IPerforceRepo, IPerforceView, changePerforceManagerSingleton} from "./perforceManager";
 import {Repo, repStore} from "./repoStore";
 import {loadingStateUpdateHandler, onAddRepoRequestHandler} from "./server";
 import {getLogger} from "./logger";
 import { setSettings, getSettings } from "./utils";
+import { isNumber } from "lodash";
```

Comment:
When I tried to use isNumber it wasn't that good. I did !isNaN in the end

Reply:
not in use - deleting this import